app/assets/stylesheets/custom.css.scss

```diff
@@ -37,6 +38,12 @@ header div h2 {
   text-align: center;
 }
 
+.thumbnail > img {
+  width:100%;
```

Comment:
Properties should be ordered height, overflow, width<br>Colon after property should be followed by one space